#request fields
float32 x
float32 y
float32 z
float32 v
---
#response fields
string res
